### PR TITLE
Utils: Add create_table method

### PR DESCRIPTION
**Pull request by sinus-x**

### CODE REVIEW

File: core/utils.py [hunk 1]
```diff
@@ -84,6 +84,47 @@ def parse_bool(string: str) -> Optional[bool]:
...
+                width = len(line[i])
+                if widths[i] < width:
+                    widths[i] = width
+
+        content: List[str] = []
+        for matrix_line in matrix:
+            line = ""
+            for i in range(len(attributes) - 1):
+                line += matrix_line[i].ljust(widths[i])
+                line += " "
```
Comment (Czechbol) [inline core/utils.py:121]: This could be just `ljust(widths[i]+1)` right?

File: core/utils.py [hunk 2]
```diff
@@ -84,6 +84,47 @@ def parse_bool(string: str) -> Optional[bool]:
...
+                line.append(str(getattr(item, attr, "")))
+            matrix.append(line)
+
+        # compute column widths
+        widths: List[int] = [0 for i in range(len(attributes))]
+        for line in matrix:
+            for i in range(len(attributes)):
+                width = len(line[i])
+                if widths[i] < width:
+                    widths[i] = width
```
Comment (Czechbol) [inline core/utils.py:114]: You could squash this into the first for loop right? Saving those precious miliseconds.

**Review by Czechbol — COMMENTED** [review]
Just minor things. I will leave the decision of changing the things or not up to you.

File: core/utils.py (hunk 2 above)
Comment (sinus-x) [reply core/utils.py:114]: I was worried it would get harder to read. Will try though.

**Comment by sinus-x** [comment]
All notes have been addressed.

**Comment by Czechbol** [comment]
Just one more thing to discuss. What about discord character limitations? Do you plan on checking if the message isn't too long after this or would it be better done here?

**Comment by sinus-x** [comment]
I was thinking about it. Here I'm assuming it just won't happen, or that it may get solved by the module. However, from a future-proof perspective, it may be easier to return a list of strings.

**Review by Czechbol — APPROVED** [review]
LGTM

**Pull request merged**